Database/Memcache/Server.hs

```diff
@@ -79,11 +76,32 @@ instance Eq Server where
 instance Ord Server where
     compare x y = compare (sid x) (sid y)
 
--- | Create a new Memcached server connection.
-newServerDefault :: ServerSpec -> IO Server
-newServerDefault ss@ServerSpec{..} = do
+-- | Configurable options when creating a 'Server'
+--
+-- At the moment, this only applies to the 'Pool' information. This can be expanded in the future.
+--
+data ServerOptions
+  = ServerOptions
+  { soNumResources :: Int
+  , soNumStripes :: Int
+#if MIN_VERSION_resource_pool(0,3,0)
+  , soKeepAlive :: Double
```

Comment:
suggestion since this is new: what about just using the `Double` version since its newer, and create a NominalDiffTime when needed by converting the double?

Reply:
I can't picture in my mind's eye how that would look. Where do we make the decision between double and nominal diff time?